Makefile

```diff
@@ -268,6 +268,11 @@ ifdef MAPSIZE
     CXXFLAGS += -DMAPSIZE=$(MAPSIZE)
 endif
 
+ifeq ($(shell git rev-parse --is-inside-work-tree),true)
```

Comment:
Still assumes user has `git`. I'm wondering if there might be a clean way to handle the cases where there is no `git` (e.g. downloaded a snapshot). Just a though though.